data/apoorv-goyal.json

```
@@ -0,0 +1,114 @@
+{
+  "name": "Apoorv Goyal",
+  "type": "personal",
+  "bio": "Full Stack Web Developer | Content Creator | Founder of **Virtual Tech School**",
+  
+  "tags": [
+    "Python",
+    "JavaScript",
+    "Linux"
+  ],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/apoorvtwts" },
+    { "icon": "FaGithub", "url": "https://github.com/apoorv-on-git" },
+    { "icon": "FaYoutube", "url": "https://www.youtube.com/apoorvgoyalmain" }
+  ],
+  "links": [
+    {
+      "group": "Socials Primary",
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/apoorvtwts",
+      "icon": "FaTwitter"
+    },
+    {
+      "group": "Socials Primary",
+      "name": "YouTube: Web Development | Mentorship",
+      "url": "https://youtube.com/apoorvgoyalmain",
+      "icon": "FaYoutube"
+    },
+    {
+      "group": "Socials Primary",
+      "name": "GitHub: Collaborate on Open Source",
+      "url": "https://github.com/apoorv-on-git",
+      "icon": "FaGithub"
+    },
+    {
+      "group": "Socials Primary",
+      "name": "LinkedIn: Let's connect",
+      "url": "https://www.linkedin.com/in/apoorv-goyal-a17103158/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "group": "Socials Primary",
+      "name": "Blogs: More of those coming",
+      "url": "https://apoorvgoyal.hashnode.dev/",
+      "icon": "FaLink"
+    },
+    {
+      "group": "Virtual Tech School Community",
+      "name": "Virtual Tech School: Static website made for teaching",
+      "url": "https://virtualtechschool.org/",
+      "icon": "FaLink"
+    },
+    {
+      "group": "Virtual Tech School Community",
+      "name": "Discord: Join Us!",
+      "url": "https://discord.gg/EYB8tQxjxH",
+      "icon": "FaDiscord"
+    },
+    {
+      "group": "Virtual Tech School Community",
+      "name": "Twitter: Follow us",
+      "url": "https://twitter.com/virtechschool",
+      "icon": "FaTwitter"
+    },
+    {
+      "group": "Virtual Tech School Community",
+      "name": "LinkedIn: Follow us",
+      "url": "https://www.linkedin.com/company/virtual-tech-school",
+      "icon": "FaDiscord"
```

You have mistakenly used the Discord icon for LinkedIn link. 
```suggestion
      "icon": "FaLinkedin"
```